packages/core/test/actor.test.ts

```diff
@@ -1229,10 +1166,9 @@ describe('actors', () => {
 
   it('should be able to spawn callback actors in (lazy) initial context', (done) => {
     const machine = createMachine({
-      types: {} as { context: { ref: CallbackActorRef<{ type: 'TEST' }> } },
+      types: {} as { context: { ref: CallbackActorRef<EventObject> } },
```

Comment:
this type is about the **receiver**, not about the `sendBack`, and `{ type: 'TEST' }` is used by `sendBack`, so the previous annotation was wrong